src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -1346,6 +1367,8 @@ void SocketTraceConnector::AppendMessage(ConnectorContext* ctx, const ConnTracke
 
   r.Append<r.ColIndex("req_msg")>(entry.req.msg);
   r.Append<r.ColIndex("resp_msg")>(entry.resp.msg);
+  r.Append<r.ColIndex("latency")>(
```

Comment:
Let's also edit the `amqp_data.pxl` pxl script and add latency as a column.

Reply:
Fixed in 1eefd2fce4466165e63761d8e189de5f6b00ce2d